package.json

```diff
@@ -28,10 +28,10 @@
   ],
   "dependencies": {
     "@prisma/client": "^4.16.2",
-    "@remix-run/css-bundle": "*",
-    "@remix-run/node": "*",
-    "@remix-run/react": "*",
-    "@remix-run/serve": "*",
+    "@remix-run/css-bundle": "^2.0.0-pre.8",
+    "@remix-run/node": "^2.0.0-pre.8",
+    "@remix-run/react": "^2.0.0-pre.8",
+    "@remix-run/serve": "^2.0.0-pre.8",
```

Comment:
We can probably leave these as `*` in this branch since we won't merge this util a stable `2.0.0` is out and that way we do't forget to change it back.  For now I've been manually running `npx upgrade-remix pre` in all my testing of templates and stacks for v2.